Web_Development/Javascript/Topics/6.Variables/README.md

```diff
@@ -0,0 +1,273 @@
+## Javascript Variables
+
+In this article you are going to learn everything about **Javascript Variables** from scratch. 
+
+Variables are "containers" for storing information JavaScript variables are used to hold values or
+expressions. A variable can have a short name, like x, or a more descriptive name, like carname.
+
+ ## Rules for declaring Javascript Variable name:
+There are some rules while declaring a JavaScript variable (also known as identifiers).
+
+   1. Name must start with a letter (a to z or A to Z), underscore( _ ), or dollar( $ ) sign.
+   2. After first letter we can use digits (0 to 9), for example value1.
+   3. JavaScript variables are case sensitive, for example x and X are different variables.
+
```

Comment:
Give examples of variable names to show which is allowed and which is not? It should cover all the examples from the rules.
For example
```js
var adam; //correct
var 1adam; //incorrect
```
Give all the examples.